app/models/transformation_mapping/vm_migration_validator.rb

```diff
@@ -132,7 +132,8 @@ def initialize(vm_name, vm, status, reason)
           "cluster"        => vm.ems_cluster.try(:name) || '',
           "path"           => vm.ext_management_system ? "#{vm.ext_management_system.name}/#{vm.v_parent_blue_folder_display_path}" : '',
           "allocated_size" => vm.allocated_disk_storage,
-          "id"             => vm.id.to_s
+          "id"             => vm.id.to_s,
+          "hardware"       => vm.hardware
```

Comment:
Are you intending to put the whole ActiveRecord hardware object as a value in the hash struct?

Reply:
We technically only need `memory_mb` from the hardware object.  Would it be better to specify the exact attribute?